tools/install_prereqs.sh

```diff
@@ -56,8 +56,15 @@ echo "Dependencies are installed at $PREFIX"
 # Install deps with rbuild
 rbuild prepare -d $PREFIX -s develop
 
-# install onnx package for unit tests
-pip3 install onnx==1.10.2 numpy==1.21.6 typing==3.7.4 pytest==6.0.1 packaging==23.0
+if ! python3 -c 'import sys; assert sys.version_info >= (3, 10)' > /dev/null; then
+  python3.8 -m pip install protobuf==3.20.3 onnx==1.10.2 numpy==1.21.6 typing==3.7.4 pytest==6.0.1 packaging==23.0
```

Comment:
You just checked for version 3.10 so you shouldn't be using 3.8.

Reply:
Removed